openspec/changes/add-hybrid-search/tasks.md

```
@@ -0,0 +1,156 @@
+# Tasks: 混合检索算法（Hybrid Search）
+
+## 任务列表
+
+### 1. 数据结构定义
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)